File: Sort/Bubble Sort/Bubble Sort.java
import java.util.Scanner; // For taking input

class Bubble_Sort
{
	public static void main(String args[])
	{
		Scanner input = new Scanner(System.in);
		
		int N;
		System.out.print("\n Array size : ");
		N = input.nextInt();

		int A[] = new int[N], temp;
		System.out.print("\n Enter elements : ");

		for(int i = 0; i < N; ++i)
			A[i] = input.nextInt();

		// Bubble Sort
		//-------------
		for(int i = 0; i < (N-1); ++i)
		{
			for(int j = i; j < (N-1-i); ++j)
			{
				if(A[j] > A[j+1])
				{
					// Swapping
					//----------
					temp = A[j];
					A[j] = A[j+1];
					A[j+1] = temp;
				}
			}
		}

		System.out.print("\n Sorted Array : ");
		for(int i = 0; i < N; ++i)
			System.out.print(A[i] + " ");
	}
}

File: Sort/Bubble Sort/Bubble Sort.cpp
#include <iostream>

using namespace std;

int main()
{
	int N;
	cout << "\n Array size : ";
	cin >> N;

	int a[N], temp;
	cout << "\n Enter elements : ";

	for(int i = 0; i < N; ++i)
		cin >> a[i];

	// Bubble Sort
	//-------------
	for(int i = 0; i < (N - 1); ++i)
	{
		for(int j = i; j < (N - 1 - i); ++j)
		{
			if(a[j] > a[j+1])
			{
				// Swapping
				//----------
				temp = a[j];
				a[j] = a[j+1];
				a[j+1] = temp;
			}
		}
	}

	cout << "\n Sorted Array : ";

	for(int i = 0; i < N; ++i)
		cout << a[i] << " ";

	return 0;
}